.svgrrc

```diff
@@ -0,0 +1,5 @@
+{
+  "replaceAttrValues": {
+    "#0A1427": "{props.fill}"
```

Comment:
The way it works, is that `#0A1427` in SVG file will be replaced by provided `fill` props. Essentially we mark given color as a template.

Please compare: https://github.com/kristerkari/react-native-svg-transformer#changing-svg-fill-color-in-js-code

Reply:
I tested this, and it looks like it could be an arbitrary string instead, to make it more clear what it is meant to do as long as we're fine with the default not being a valid colour -- which wouldn't be left in place for themed items.

Reply:
Very interesting, I'll check that.

Reply:
Confirmed. I'll use `fillColor` as the template string.